src/core/sys/osx/sys/mman.d

```diff
@@ -2,7 +2,7 @@
  * $(RED Deprecated. Use $(D core.sys.darwin.sys.mman) instead. This module
  *       will be removed in June 2018.)
  *
- * D header file for FreeBSD
+ * D header file for OSX
```

Comment:
This change doesn't look related to DragonFly?

Reply:
Yep, just a random comment fix, no problem with including it.

Reply:
Reverted
@joakim-noah  Woops :-)

Reply:
You can just remove that last revert commit.